app/views/content/whats_new.nl.html.md

```diff
@@ -1,6 +1,20 @@
 # Wat is er nieuw in het Energietransitiemodel?
 ___
 
+# Februari 2023
+
+## Nieuwe feature voor technologieën voor elektriciteitsopslag
+
+In de sectie *Flexibiliteit* → **[Opslag elektriciteit](/scenario/flexibility/flexibility_storage/behaviour-of-storage-technologies)** kun je per opslagtechnologie bepalen of je het gedrag wil laten bepalen door een prognose-algoritme.
```

Comment:
Nitpick: the verb "bepalen" is used twice in this sentence. I would change one of those to another verb. I leave it up to you if you think it's worth to adjust this, @mabijkerk :)